implementations/rust/ockam/ockam_node/src/router/stop_worker.rs

```diff
@@ -41,14 +41,6 @@ pub(super) async fn exec(
         }
     };
 
-    // Remove all secondary addresses
-    for addr in record.address_set().iter() {
```

Comment:
I don't get why removing the aliases is no longer necessary

Reply:
It's still needed, but it was removed twice. One time here and one time in `InternalMap::free_address` after shutdown acknowledgment was received from a Worker. I think there are a lot of similar surprises in `ockam_node` crate...